src/main/java/Duke.java

```diff
@@ -1,10 +1,110 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final int MAX_TASKS_COUNT = 100;
+    private static Task[] tasks = new Task[MAX_TASKS_COUNT];
+    private static int indexOfTasks = 0;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
+        String name = "Duke";
+        String input;
+        Scanner reader = new Scanner(System.in);
+
+        String logo = " ____        _\n"
+                + "|  _ \\ _   _| | _____\n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("\tHello from\n" + logo);
+
+        printLine();
+        welcome(name);
+        printLine();
```

Comment:
The print codes are short and neat, but perhaps it one way to improve general readability would be to refractor these into a method such as "printWelcomeMessage()"